osu.Game/Beatmaps/IBeatmapDifficultyInfo.cs

```diff
@@ -55,13 +70,20 @@ public interface IBeatmapDifficultyInfo
         static double DifficultyRange(double difficulty, double min, double mid, double max)
         {
             if (difficulty > 5)
-                return mid + (max - mid) * (difficulty - 5) / 5;
+                return mid + (max - mid) * DifficultyRange(difficulty);
```

Comment:
Not sure why only one branch uses this helper when two could:

```diff
diff --git a/osu.Game/Beatmaps/IBeatmapDifficultyInfo.cs b/osu.Game/Beatmaps/IBeatmapDifficultyInfo.cs
index af943b62e7..19944afa1c 100644
--- a/osu.Game/Beatmaps/IBeatmapDifficultyInfo.cs
+++ b/osu.Game/Beatmaps/IBeatmapDifficultyInfo.cs
@@ -72,7 +72,7 @@ static double DifficultyRange(double difficulty, double min, double mid, double
             if (difficulty > 5)
                 return mid + (max - mid) * DifficultyRange(difficulty);
             if (difficulty < 5)
-                return mid - (mid - min) * (5 - difficulty) / 5;
+                return mid + (mid - min) * DifficultyRange(difficulty);
 
             return mid;
         }

```

Reply:
I had a feeling this was possible but brain was off at this point.